tests/golden/defaults/appcat/appcat/sli_exporter/90_VSHNNextcloud_Opsgenie.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    syn: 'true'
+    syn_component: appcat
+    syn_team: schedar
+  name: vshn-vshnnextcloud-new-sla
+  namespace: appcat-slos
+spec:
+  groups:
+    - name: appcat-vshnnextcloud-sla-target
+      rules:
+        - alert: vshn-vshnnextcloud-new-sla
+          annotations:
+            summary: '{{$labels.service}} {{$labels.name}} down in {{$labels.namespace}}'
+            title: '{{$labels.service}} {{$labels.name}} down in {{$labels.namespace}}'
+          expr: rate(appcat_probes_seconds_count{reason!="success", service="VSHNNextcloud",
+            ha="false", maintenance="false"}[5m]) > 0.2 and rate(appcat_probes_seconds_count{reason!="success",
+            service="VSHNNextcloud", ha="false", maintenance="false"}[1m]) > 0.75
```

Comment:
nextcloud sli probe doesn't exist yet, it waits for `alerting/metricts maturity level`